### ISSUE TITLE
Mobile

### ISSUE THREAD

**Issue by ksami** [ksami/muddy#2]
Support mobile? Or leave them?

**Comment by MasamuneGoro** [comment]
Does it affect the way we code? Ideally support mobile.

**Comment by ksami** [comment]
Yes, need a completely new frontend for mobile including new ways of input and/or displaying output.
